MetaMask/metamask-extension
# refactor: remove whats new notification for staking

## **Description**

Removes what's new notification for direct staking within MetaMask. This notification is being removed because it has been there for some time now and our team has been asked to remove it by May 9th

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/24401?quickstart=1)

## **Related issues**

Fixes: [STAKE-689](https://consensyssoftware.atlassian.net/browse/STAKE-689)

## **Manual testing steps**

1. Install or update extension
2. View what's new modal
3. Confirm that staking notification is no longer there 

## **Screenshots/Recordings**

### **Before**

https://www.loom.com/share/bd5aee8a123e4e1c8d76bbd89a22c32b

### **After**

https://www.loom.com/share/73c1e2924271431c8a98c8aed07da4dc

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
